sources/packages/backend/apps/api/src/route-controllers/announcements/models/announcements.dto.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Announcements detail DTO. This is used for view only purpose.
+ */
+export class AnnouncementsAPIOutDTO {
+  message_title: string;
+  message: string;
+  start_date: Date;
+  end_date: Date;
+  target: string;
```

Comment:
As mentioned by @sh16011993 please use camel case for properties e.g `startDate`

Reply:
updated